travharv/config_build.py

```diff
@@ -187,17 +184,17 @@ def _make_path_parts(self, assert_path):
         """
         Make the path parts by splitting the path string on regex expression
         """
-        REGEXP = r"(?:\w+:\w+|<[^>]+>)"
+        REGEXP = r"\s*/\s*(?![^<]*>)"  # TODO - define the regex expression
```

Comment:
why not use the splitter already in the helper.py?
in fact, that one could deliver the unjoined (and stil resolved) parts as easily !